src/components/YourCompany/CompanyName/CorrectNameToNumber.vue

```diff
@@ -1,12 +1,15 @@
 <template>
   <v-form id="correct-name-to-number-form" lazy-validation>
-    <v-layout row>
+    <v-row no-gutters>
+      <v-col>
         <v-checkbox
           v-model="correctToNumbered"
           id="correct-name-to-number-checkbox"
+          class="mb-n5"
```

Comment:
Negative margin used here to "shorten" the height of this component. (Components below this will move into this space.)